recipe/meta.yaml

```diff
@@ -8,30 +8,33 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6e2fb792ca022f900f374f9659fd3e7c530cd4df7d3b7e84be889093b487639f
+  sha256: deb2eb9e9119d911debe23edcec8ea68a2cd70c9e3f753c96aaf4a86ca497e7e
 
 build:
   number: 0
+  skip: true  # [py<36]
   script:
-    # So Cython regenerates it:
-    - rm -rf _line_profiler.c  # [unix]
-    - del /s /q _line_profiler.c  # [win]
-    - {{ PYTHON }} setup.py install --old-and-unmanageable
+    # We can't use CMAKE_GENERATOR implicitly via conda_build_config.yaml, yet,
+    # due to https://github.com/AnacondaRecipes/aggregate/pull/182
+    - set "CMAKE_GENERATOR=Ninja"  # [win]
+    - export CMAKE_GENERATOR=Ninja  # [not (win or (osx and arm64))]
+    - export CMAKE_OSX_ARCHITECTURES="arm64"  # [osx and arm64]
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
-    - kernprof=kernprof:main
-  # package is required on linux-s390x
-  skip: True  [s390x]
+    - kernprof = kernprof:main
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - cmake
-    - make      # [unix]
+    - ninja
```

Comment:
Upstream uses ninja >= 1.10.2 (https://github.com/pyutils/line_profiler/blob/v4.0.3/requirements/build.txt#L7).

Reply:
For the build section, conda-build always will pick up the latest version of a package. And our current version is 1.10.2 so it's not needed to pin it here